src/openms/source/FORMAT/SqliteConnector.cpp

```diff
@@ -175,11 +175,11 @@ namespace OpenMS
         return false;
       }
 
-      template <> bool extractValue<int>(int* dst, sqlite3_stmt* stmt, int pos) //explicit specialization
+      template <> bool extractValue<int>(Int32* dst, sqlite3_stmt* stmt, int pos) //explicit specialization
       {
         if (sqlite3_column_type(stmt, pos) != SQLITE_NULL)
         {
-          *dst = sqlite3_column_int(stmt, pos);
+          *dst = sqlite3_column_int(stmt, pos); // sqlite3_column_int returns 32bit integers
```

Comment:
not sure if this is the right fix... int should be 4 byte already

runtime error: store to misaligned address 0x7ffe16de1061 for type 'int', which requires 4 byte alignment